cli/util/convert.go

```diff
@@ -15,6 +16,13 @@ import (
 // NewCommands returns util commands for neo-go CLI.
 func NewCommands() []cli.Command {
 	txDumpFlags := append([]cli.Flag{}, options.RPC...)
+	txCancelFlags := append([]cli.Flag{
+		flags.AddressFlag{
+			Name:  "address, a",
+			Usage: "address to use as transaction signee (and gas source)",
```

Comment:
This flag could also be exported and reusable (`GetAccountFromContext` relies on it), but if we have it locally in the `util` package, then let's adjust help message a bit: s/`as transaction`/`as conflicting transaction`